Plots/Masking/NCL_mask_5.py

```diff
@@ -0,0 +1,142 @@
+    - Using draw order resources to mask areas in a plot
+    - Adding a label bar
+    - Implementing best practices when choosing a color scheme 
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/mask_5.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/mask_5_lg.png
+                         
+"""
+
+# ###############################################################################
+# Import packages:
+
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+import matplotlib.patches as mpatches
+
+from geocat.viz import util as gvutil
+import geocat.datafiles as gdf
+
+###############################################################################
+#Read in Data
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/atmos.nc"), decode_times=False)
+t = ds.TS.isel(time=0)
+
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
+wrap_t = gvutil.xr_add_cyclic_longitudes(t, "lon")
+
+###############################################################################
+#Plot:
+
+fig = plt.figure(figsize=(10, 15))
+
+def Plot(row, col, pos, title):
+    """
+    Helper function to create two similar plots where subplot position
+    and title can all be customized on the same style
+    map projection.
+
+    Args:
+
+        row (:class: 'int'):
+            number of rows necessary for subplotting of visualizations
+        col (:class: 'int'):
+            number of columns necessary for subplotting
+        pos (:class: 'int'):
+            position of visualization in m x n subplot
+        title (:class: 'str'):
+            title of graph in format "Title"
+     """
+
+    # Generate axes, using Cartopy, drawing coastlines, and adding features
+    projection = ccrs.PlateCarree()
+    ax = plt.subplot(row, col, pos, projection=projection)
+    ax.coastlines(linewidths=0.5)
+    ax.add_feature(cfeature.LAND, color="green")
+    ax.add_feature(cfeature.LAKES, color="plum")
+    ax.add_feature(cfeature.OCEAN, color="blue")
+
+    # Create label names and define colors for the legend
+    land = mpatches.Rectangle((0, 0), 1, 1, facecolor="green")
+    lakes = mpatches.Rectangle((0, 0), 1, 1, facecolor="plum")
+    ocean = mpatches.Rectangle((0, 0), 1, 1, facecolor="blue")
+
+    labels = ['Land ', 'Lakes', 'Ocean']
+
+    plt.legend([land, lakes, ocean], labels,
+               loc='lower center', bbox_to_anchor=(0.5, -0.2), ncol=3)
+
+    # Use geocat.viz.util convenience function to set titles and labels without calling several matplotlib functions
+    gvutil.set_titles_and_labels(
+        ax,
+        maintitle=title,
+        maintitlefontsize=14)
+
+# #Plot first color map
+Plot(2, 3, (1,3), "land sea mask using 'atmos.nc'")
```

Comment:
I'm having a hard time figuring out why the `row` `col` and `pos` need to be these numbers. I'm looking at the `subplot` documentation and I'm still not sure. Can you add an elaborating comment or make some changes to make the code more intuitive.

Reply:
Originally, my IDE was making the 2x1 subplot look like this:
![image](https://user-images.githubusercontent.com/43652875/94580871-a6cf6a80-0237-11eb-9772-032f8e35842f.png)
where the size of the two subplots were not the same and there was overlap that wasn't going to be fixable with position calls. So those numbers were there because that was the only way to get the plots to be formatted correctly. But as of this morning, my IDE has decided it wants to make the 2x1 subplot correctly, and I have updated with that fix. In the IDE and after testing with sphinx the output is this:
![image](https://user-images.githubusercontent.com/43652875/94581263-1b0a0e00-0238-11eb-8d58-151f6c88ab6e.png)